model/config.go

```diff
@@ -757,20 +759,20 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 
 type ClusterSettings struct {
 	Enable                             *bool   `access:"environment,write_restrictable"`
```

Comment:
This one doesn't have the `cloud_restrictable` tag because is used [here](https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/admin_definition.jsx#L2026) and fails if is not present